components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -135,38 +138,39 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
         this.isMinorV1 = version === 'v1';
       });
 
-    this.store.select(assignableProjects)
-      .subscribe((assignable: ProjectsFilterOption[]) => {
-        // compiles a list of all available projects
+    combineLatest([
+      this.store.select(assignableProjects),
+      this.store.select(optionsLoadingStatus),
+      this.store.select(v1TeamFromRoute),
+      this.store.select(v2TeamFromRoute),
+      this.store.select(iamMajorVersion)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      map(([assignable, pStatus, v1Team, v2Team, major]:
+        [ProjectsFilterOption[], EntityStatus, Team, Team, IAMMajorVersion]) => {
+        this.isMajorV1 = major === 'v1';
+        return [assignable, pStatus, (this.isMajorV1 ? v1Team : v2Team)];
```

Comment:
🤔 If we used an object (and an interface) here, we could reduce the duplications a bit, I think...like this:
```typescript
map(([assignable, pStatus, v1Team, v2Team, major]: [ProjectsFilterOption[], EntityStatus, Team, Team, IAMMajorVersion]) => {
    this.isMajorV1 = major === 'v1';
    return { assignable, pStatus, team: (this.isMajorV1 ? v1Team : v2Team) };
}),
filter({ pStatus } => pStatus !== EntityStatus.loading), // ignore while loading
filter({ team }) => !!team), // ignore while undefined
map({ assignable, team) => {
  this.team = team;
  assignable.forEach(...)
})
```
(I've omitted the interface, I think we'd want a type there somewhere...) -- Not sure if this is better, just pointing out an alternative (which I haven't tested ⚠️)